elytron/src/test/resources/org/wildfly/extension/elytron/tls-oracle13plus.xml

```diff
@@ -0,0 +1,86 @@
+<!-- for needs of SaslTestCase and KeyStoresTestCase -->
+<subsystem xmlns="urn:wildfly:elytron:7.0" default-ssl-context="ClientSslContextNoAuth">
+    <providers>
+        <provider-loader name="ManagerProviderLoader" class-names="sun.security.ssl.SunJSSE"/>
```

Comment:
so that is only one change compared to tls-sun.xml. Could be tls-sun.xml parametrized?

Reply:
I'm not that familiar with WildFly configuration mock test support @mchoma .
@darranl @bstansberry @kabir is it possible to parametrize subsystem test configurations
based on condition comparing 'java.specification.version' JVM property?

Reply:
If the attribute supports expressions you can put an expression in the xml and then set a system property before running the test; e.g. the code you have now that's selecting which config file could instead set a property.

I haven't looked, but these may not support expressions. Historically we didn't for 'class name' attributes. For no great reason really. This is the first good use case for using them in that kind of attribute that I've seen.

Reply:
@mchoma I'm not going to investigate this. Let's leave the proposed changes as they are - it's all just about tests.

Reply:
Ok I dont insist. Setting /elytron/provider-loader/class-names/expressions-allowed = true would help. But as Brian said it is everywhere else false. So we should be consistent.

Reply:
I don't mind changing an expressions-allowed setting if that's what Darran/Farah/Jeff Mesnil prefer. There's really not a good reason for having those as false and if there's a reasonable, understandable use case for true for a particular attribute I don't think we have to enforce a rule against it.

I agree a change like that is beyond the scope of this PR though.

Reply:
Thinking again. Shouldn't it work without provider-loader name="ManagerProviderLoader" by default?. So by default using just system providers? Or is specifying provider loader what is tested here?

Reply:
Off topic for this PR but when we started defining attributes with expression support enabled and disabled it was primarily to avoid references to other parts of the model being handled as an expression, but at the time we didn't have support for capabilities.

I wonder if we are getting nearer to a point where expression support is enabled unless the attribute is a capability reference.